.travis.yml

```diff
@@ -2,14 +2,10 @@ language: ruby
 sudo: false
 script: rake
 rvm:
-  - 2.2.0
-  - 2.1.5
-  - 2.0.0
-  - 1.9.3
-  - jruby-1.7.9-d19 # fails after 1.7.10 for some reason
-#  - jruby-19mode # JRuby in 1.9 mode
+  - 2.2.3
```

Comment:
No, revs now uses revs-indexing-service boxes, with the new StanfordSync stack (discovery_indexer, base_indexer, etc.)  The old revs code is still on the harvestdor-xxx boxes and probably still works, but it is not updated, used or maintained anymore.